x-pack/plugins/reporting/server/lib/event_logger/index.ts

```diff
@@ -16,7 +13,5 @@ export enum ActionType {
   SAVE_REPORT = 'save-report',
   RETRY = 'retry',
   FAIL_REPORT = 'fail-report',
-}
-export function registerEventLogProviderActions(eventLog: IEventLogService) {
-  eventLog.registerProviderActions(PLUGIN_ID, Object.values(ActionType));
+  EXECUTE_ERROR = 'execute-error',
```

Comment:
This change is not related to this PR. It's a discovered oversight that should have gone into the first Event Log PR for Reporting.